cfpb/hmda-frontend
# Q659-1: Highlight usage of 888 or 88888 in fields where 8888 == NA

Closes #1199 
## Screenshots
<img width="1108" alt="2022_Q1~Q659-1" src="https://user-images.githubusercontent.com/2592907/142946722-2813a745-e400-4bba-8e88-16744205dc43.png">

## Testing via Local Platform
- Note: This is a new edit for 2022
- Open filing on the Frontend for 2022-Q1 via `src/common/constants/dev-config.json`
  <img width="322" alt="config" src="https://user-images.githubusercontent.com/2592907/142946988-ac6a1ea8-6172-4fbc-9ef2-03c69609c268.png">
- Submit test file `cypress/fixtures/2022-Q1-FRONTENDTESTBANK9999-Q659-1.txt` for `2022-Q1`
- Observe `888`/`88888` highlighted in Q659-1 listing while `8888` left unhighlighted.
